README.md

```diff
@@ -0,0 +1,1116 @@
+sed -i '' 's%13944328343563%6013944328343563%g' genesis.json
+# Increase the validator power by 6,000,000,000
+sed -i '' 's%"power": "13944328"%"power": "6013944328"%g' genesis.json
+
+# fix last_total_power
+# Increase total amounts of bonded tokens recorded during the previous end block by 6,000,000,000
+sed -i '' 's%"194616038"%"6194616038"%g' genesis.json
+
+# fix total supply of uatom
+sed -i '' 's%277834757180509%6277834757180509%g' genesis.json
+
+# fix balance of bonded_tokens_pool module account
+#
+# module account for recording Binance staking(validator 2)'s received delegations:
+# cosmos1fl48vsnmsdzcv85q5d2q4z5ajdha8yu34mf0eh
+# Increase the delegation account by 6,000,000,000,000,000
+sed -i '' 's%194616098248861%6194616098248861%g' genesis.json
+```
+
+#### Modify some gov parameters for test efficiency
+
+Change the minimum deposit amount, quorum, threshold, and voting period.Those changes can get the user account (who is also a delegator)'s vote pass when voting for the upgrade proposal.
+```shell
+# min deposition amount
+sed -i '' 's%"amount": "64000000",%"amount": "1",%g' genesis.json
+#   min voting power that a proposal requires in order to be a valid proposal
+sed -i '' 's%"quorum": "0.400000000000000000",%"quorum": "0.000000000000000001",%g' genesis.json
+# the minimum proportion of "yes" votes requires for the proposal to pass
+sed -i '' 's%"threshold": "0.500000000000000000",%"threshold": "0.000000000000000001",%g' genesis.json
+# voting period 
+sed -i '' 's%"voting_period": "1209600s"%"voting_period": "60s"%g' genesis.json
+```
+### Init the chain
+#### Setup the environmental variables
+```shell
+export EXPORTED_GENESIS=genesis.json 
+export BINARY=gaiad 
+export CHAIN_ID=test 
+export CHAIN_DIR=data 
+
+export VAL_1_CHAIN_DIR=$CHAIN_DIR/$CHAIN_ID/val1 
+export VAL_2_CHAIN_DIR=$CHAIN_DIR/$CHAIN_ID/val2 
+export VAL_1_KEY_NAME="val1" 
+export VAL_2_KEY_NAME="val2" 
+export VAL_1_MONIKER="val1" 
+export VAL_2_MONIKER="val2" 
+
+export USER_CHAIN_DIR=$CHAIN_DIR/$CHAIN_ID/val2
+export USER_MNEMONIC="junk appear guide guess bar reject vendor illegal script sting shock afraid detect ginger other theory relief dress develop core pull across hen float"
+export USER_KEY_NAME="user"
+```
+#### Init the chain and setup the user account
+```shell
+$BINARY config chain-id test --home $VAL_1_CHAIN_DIR 
+$BINARY config keyring-backend test --home $VAL_1_CHAIN_DIR 
+$BINARY config broadcast-mode block --home $VAL_1_CHAIN_DIR 
+
+$BINARY config chain-id test --home $VAL_2_CHAIN_DIR 
+$BINARY config keyring-backend test --home $VAL_2_CHAIN_DIR 
+$BINARY config broadcast-mode block --home $VAL_2_CHAIN_DIR 
+
+# Validator 1
+$BINARY init test --home $VAL_1_CHAIN_DIR --chain-id=$CHAIN_ID
+# Validator 2
+$BINARY --home $VAL_2_CHAIN_DIR init test --chain-id=$CHAIN_ID
+#user
+echo $USER_2_MNEMONIC | $BINARY --home $USER_2_CHAIN_DIR keys add $USER_2_KEY_NAME --recover --keyring-backend=test
```

Comment:
`$USER_2_MNEMONIC` is never set in this shell sequence

@yaruwangway can you confirm which one is used here?

Reply:
Hi, I changed `$USER_2` to `$USER`, please find in the new commit fabf471ec36658f2bee84a540c8efcdc00cee7f9